ethereum/contracts/Bridge.sol

```diff
@@ -114,6 +136,11 @@ contract Bridge is IBridge, AccessControl {
         _;
     }
 
+    modifier verifyReverseSwapAmount(uint256 amount) {
+        require(amount <= swapMax, "Amount exceeds upper limit");
```

Comment:
@pbukva so you only check the max?

Reply:
yes, min (or fee) is  irrelevant in the sense that primary purpose of the this check is to prevent massive rug pull when relayer key gets exposed.
Min and fee are properly handled in the reverseSwap and refund... methods (if the `swap.amount is < swapMin` or `swap.amount is < swapFee`), both methods will handle that correctly = they will execute successfully withdrawing `swap.amount` as swapFee (we (=Fetch.ai) will loose there a little bit on Tx fee, but that is fine), and user will get nothing.

Reply:
LGTM.